src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -106,23 +118,31 @@ private void OnWorkspacePropertyChanged(object sender, PropertyChangedEventArgs
         /// <param name="ws">workspace model</param>
         internal void DependencyRegen(WorkspaceModel ws)
         {
+            this.RestartBanner.Visibility = Visibility.Hidden;
             var packageDependencies = ws.NodeLibraryDependencies.Where(d => d is PackageDependencyInfo);
 
             if (packageDependencies.Any(d => d.State != PackageDependencyState.Loaded))
             {
                 HasDependencyIssue = true;
             }
 
-            PackageDependencyTable.ItemsSource = packageDependencies.Select(d => new PackageDependencyRow(d as PackageDependencyInfo));
+            if (packageDependencies.Any(d => d.State != PackageDependencyState.RequiresRestart))
```

Comment:
discussed in person - this seems backwards but it's working... requires some investigation..